src/components/FeatureSetsPanel/FeatureSetsPanelTargetStore/FeatureSetsPanelTargetStoreView.js

```diff
@@ -166,51 +184,58 @@ const FeatureSetsPanelTargetStoreView = ({
               {otherKindData.kind === 'parquet' && (
                 <CheckBox
                   item={{ id: 'partitioned', label: 'Partition' }}
-                  onChange={id => triggerOfflinePartition(id, 'other')}
+                  onChange={id =>
+                    triggerOfflinePartition(id, 'externalOffline')
+                  }
                   selectedId={otherKindData.partitioned}
                 />
               )}
               {otherKindData.partitioned && (
                 <PartitionFields
                   data={otherKindData}
-                  partitionColsOnBlur={() => handlePartitionColsOnBlur('other')}
+                  partitionColsOnBlur={() =>
+                    handlePartitionColsOnBlur('externalOffline')
+                  }
                   rangeOnChange={value =>
-                    handleKeyBucketingNumberChange(value, 'other')
+                    handleKeyBucketingNumberChange(value, 'externalOffline')
                   }
                   setData={setOtherKindData}
                   timePartitioningGranularityChange={value =>
-                    handleTimePartitioningGranularityChange(value, 'other')
+                    handleTimePartitioningGranularityChange(
+                      value,
+                      'externalOffline'
+                    )
                   }
                 />
               )}
             </div>
           )}
         </div>
+        {!selectedTargetKind.length && (
```

Comment:
would be better to read

```suggestion
        {selectedTargetKind.length === 0 && (
```